theme/docs.less

```diff
@@ -120,6 +120,15 @@
     .ui.primary.button p {
         color: white;
     }
+
+    #sidedocs-back-button {
+        margin-top: 1.25rem;
+        margin-left: 1.25rem;
```

Comment:
`cursor: pointer;` maybe? From your gif it looks like currently the cursor remains as `text`